KDIC/BlogTableCell.swift

```diff
@@ -1,23 +1,19 @@
-//
-//  BlogTableCell.swift
-//  KDIC
-//
-//  Created by Shaun Mataire on 3/1/16.
-//  Copyright © 2016 Colin Tremblay. All rights reserved.
-//
-
 import UIKit
 
 class BlogTableCell: UITableViewCell {
     
+  
     @IBOutlet weak var blogPostTitle: UILabel!
```

Comment:
Remember that here we're naming the label, not the text being displayed by the label. Just `title` would suffice, or maybe `titleLabel`?